### PR TITLE
Explain filter with nested property

### PR DESCRIPTION
**Pull request by jannyHou**
Connect to https://github.com/strongloop/loopback/issues/517

Loopback support nested property filter in mongodb, memory, cloudant connectors, details seehttps://github.com/strongloop/loopback/issues/517#issuecomment-275532100

This pr explains to users about which connectors support the feature and also includes an example shows what a nested property filter looks like.

### CODE REVIEW

**Comment by jannyHou** [comment]
@kjdelisle PTAL. Thanks.

File: pages/en/lb3/Querying-data.md [hunk 1]
```diff
@@ -312,3 +312,29 @@ As a LoopBack user this is a pretty powerful thing.
...
+        tag: String,
+      }
+    ]
+  }
+});
+```
+
+users can do a nested query like `User.find({where: {'address.tags.tag': 'business'}}`.
+
+We currently don't support such a filter in SQL connectors.
```
Comment (crandmck) [inline pages/en/lb3/Querying-data.md]: Wording nitpick:  "Data source connectors for relational databases don't support filtering nested properties."

File: pages/en/lb3/Querying-data.md [hunk 2]
```diff
@@ -312,3 +312,29 @@ As a LoopBack user this is a pretty powerful thing.
 Typically, you will have learned how to write some complex queries using the `find()` filter syntax; before you would need to figure out how to do the
 same thing in JavaScript (perhaps using a library such as underscore). Now with the `loopback-filters` module, in your client application you can re-use
 the same exact filter object you were sending to the server to filter the database without having to interact with a LoopBack server at all.
+
+### Filtering nested properties
+
+Loopback supports filtering nested properties in 3 NoSQL connectors: Mongodb, Cloudant, Memory.
```
Comment (crandmck) [inline pages/en/lb3/Querying-data.md]: Please spell out "three".

**Review by crandmck — CHANGES_REQUESTED** [review]

**Comment by crandmck** [comment]
Thx LGTM

**Pull request merged**